src/gfx/math.hpp

```diff
@@ -40,6 +40,11 @@ template <size_t Alignment> inline constexpr size_t alignTo(size_t size) {
   }
 }
 
+template <typename T> inline bool isWithinRange(T val, T target, T tolerance = 0.05f) {
+  T delta = std::abs(val - target);
+  return delta > -tolerance && delta < tolerance;
```

Comment:
Is that to allow a negative tolerance? Wouldn't it be better to assert that tolerance is always strictly positive (and even different from zero).

Reply:
Overlooked that, removed the abs

Reply:
Removed the abs